internal/providers/terraform/azure/testdata/automation_dsc_configuration_test/automation_dsc_configuration_test.golden

```diff
@@ -0,0 +1,20 @@
+
+ Name                                                   Monthly Qty  Unit              Monthly Cost 
+                                                                                                    
+ azurerm_automation_dsc_configuration.fiveNodes                                                     
+ └─ Non-azure config nodes                                        5  nodes                   $30.00 
+                                                                                                    
+ azurerm_automation_dsc_configuration.without_usage                                                 
+ └─ Non-azure config nodes                           Monthly cost depends on usage: $6.00 per nodes 
+                                                                                                    
+ azurerm_automation_dsc_configuration.zeroNodes                                                     
+ └─ Non-azure config nodes                                        0  nodes                    $0.00 
+                                                                                                    
+ PROJECT TOTAL                                                                               $30.00 
+
+----------------------------------
+To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file
+
+1 resource type wasn't estimated as it's not supported yet.
+Please watch/star https://github.com/infracost/infracost as new resources are added regularly.
+1 x azurerm_automation_account
```

Comment:
Looks like it needs in update, as azurerm_automation_account should not be skipped